Sim_To_DuT_Interface/main.cpp

```diff
@@ -70,7 +71,8 @@ int main() {
     "Width",
     "current",
     "origin"};
-
+    config.periodicTimerEnabled = true;
+    config.periodicOperations = {{ "Test", 1000 }};
 
     sim_interface::dut_connector::rest_dummy::RESTDummyConnector restDummyConnector(interface.getQueueDuTToSim(), config);
     /*
```

Comment:
Wenn ich Zeile 78-87 Auskommentiere, erhalte ich im Rest Dummy DuT periodisch das Event (Indicator Right, xyz).
Erwarten würde ich aber das Event (Test, Test). Das Test Event hast du ja in Zeile 75 Konfiguriert, dieses kommt aber nur einmal ganz am Anfang.
Habe ich da was Falsch verstanden oder wie ist das zu verstehen?

Reply:
War ein Bug. Die Referenz des Events wurde überschrieben. Sollte jetzt gehen. (zumindest bei mir :D )

Reply:
Jetzt geht es